ams/api/src/main/java/com/netease/arctic/ams/api/client/OptimizingClientPools.java

```diff
@@ -46,6 +46,7 @@ private static ThriftClientPool<OptimizingService.Client> buildClient(String url
     poolConfig.setFailover(true);
     poolConfig.setMinIdle(CLIENT_POOL_MIN);
     poolConfig.setMaxIdle(CLIENT_POOL_MAX);
+    poolConfig.setMaxTotal(-1);
```

Comment:
Can you change the `CLIENT_POOL_MAX` to `CLIENT_POOL_MAX_IDLE` and `CLIENT_POOL_MIN` to `CLIENT_POOL_MIN_IDLE` to make them more consistent with their own semantics？

Reply:
Changed in the new commit.